pycqed/measurement/waveform_control/segment.py

```diff
@@ -27,6 +27,7 @@ class Segment:
     def __init__(self, name, pulse_pars_list=[]):
         self.name = name
         self.pulsar = ps.Pulsar.get_instance()
+        self.unresolved_pulses_raw = []
```

Comment:
As already discussed orally, it is not clear whether it is desirable to have two versions of unresolved pulses. If it is desirable, then:
- it is necessary to fix __repr__, which relies on unresolved_pulses (I already had an incident on the S17 branch where I was wondering why no pulses had been created until I realized that repr just kept them hidden from me.)
- maybe a better naming scheme would be unresolved_pulses and resolved_pulses (instead of unresolved_pulses_raw and unresolved_pulses)